pkgs/applications/misc/j4-dmenu-desktop/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchFromGitHub, cmake, dmenu }:
+
+stdenv.mkDerivation rec {
+  name    = "j4-dmenu-desktop-${version}";
+  version = "2.14";
+
+  src = fetchFromGitHub {
+    owner  = "enkore";
+    repo   = "j4-dmenu-desktop";
+    rev    = "r${version}";
+    sha256 = "14srrkz4qx8qplgrrjv38ri4pnkxaxaq6jy89j13xhna492bq128";
+  };
+
+  patchPhase = ''
+    substituteInPlace ./src/Main.hh \
+      --replace "\"dmenu -i\"" "\"${dmenu}/bin/dmenu -i\""
+  '';
+
+  buildInputs = [ cmake dmenu ];
```

Comment:
I just noticed that by removing `dmenu` from inputs, it don't get in the closure, is this fine?

```
nix-store --query --requisites /nix/store/q0yfp62pp3f7m6j6w1qrdsmalfm6gk2w-j4-dmenu-desktop-2.14
/nix/store/zsk34dk2wdzkryllmhfyh1bqkp77lsn6-glibc-2.23
/nix/store/f69rq0mmyp5mycgr6h8x0l252vgmq5vd-gcc-5.3.0-lib
/nix/store/q0yfp62pp3f7m6j6w1qrdsmalfm6gk2w-j4-dmenu-desktop-2.14
```